Shopist/Shopist/Scenes/MasterViewController.swift

```diff
@@ -0,0 +1,39 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import UIKit
+
+final class MasterViewController: ListViewController {
+    private var categories = [Category]()
+
+    override func fetch(with api: API) {
+        api.getCategories { result in
+            switch result {
+            case .success(let categories):
+                self.categories = categories
+            case .failure(let error):
+                print(error)
+            }
+            self.reload()
+        }
+    }
+
+    override func config(cell: Cell, for index: Int) {
+        let category = categories[index]
+        cell.text = "\(category.id): \(category.title)"
+        cell.setImage(url: category.cover)
```

Comment:
Presenting category ID looks weird. Let's display only the category name as on Android:

![ref1](https://user-images.githubusercontent.com/2358722/89170109-7b096e80-d57f-11ea-9a8e-0d4bdcf55e16.png)